packages/front-end/components/Experiment/AnalysisSettingsBar.tsx

```diff
@@ -545,6 +545,15 @@ function isDifferentStringArray(
   if (val1.length !== val2.length) return true;
   return val1.some((v) => !val2.includes(v));
 }
+function parentStringDoesNotContainAllChildrenValues(
```

Comment:
should be `parentStringArray...` (array). But consider something like `isStringArrayMissingElements()` or something...